hints/distributions/CDMC.json

```
@@ -0,0 +1,47 @@
+{
+  "hints_per_stone": 1,
+  "banned_stones": [
+    "Lanayru Sand Sea - Gossip Stone in Shipyard",
+    "Temple of Time - Gossip Stone in Temple of Time Area"
+  ],
+  "added_locations": [
+    {"location": "Upper Skyloft - Cawlin's Letter", "type": "sometimes"},
+    {"location": "Upper Skyloft - Fledge's Crystals", "type": "always"},
+    {"location": "Central Skyloft - Item in Bird Nest", "type": "sometimes"},
+    {"location": "Faron Woods - Item behind Bombable Rock", "type": "sometimes"},
+    {"location": "Lanayru Sand Sea - Skipper's Retreat - Chest in Shack", "type": "always"},
+    {"location": "Skyview - Rupee on Spring Pillar", "type": "sometimes"},
+    {"location": "Earth Temple - Rupee in Lava Tunnel", "type": "sometimes"},
+    {"location": "Ancient Cistern - Bokoblin", "type": "sometimes"},
+    {"location": "Fire Sanctuary - Mogma Mitts", "type": "sometimes"}
+  ],
+  "removed_locations": [
+    "Thunderhead - Isle of Songs - Din's Power",
+    "Faron Woods - Chest inside Great Tree",
+    "Lake Floria - Lake Floria Chest",
+    "Eldin Volcano - Digging Spot behind Boulder on Sandy Slope",
+    "Lanayru Desert - Secret Passageway Chest",
+    "Lanayru Desert - Fire Node - Shortcut Chest",
+    "Skyview - Chest behind Three Eyes",
+    "Earth Temple - Chest in West Room",
+    "Ancient Cistern - Chest in East Part",
+    "Sandship - Boss Key Chest",
+    "Lanayru Gorge - Lanayru Soth",
```

Is this location not renamed?
```suggestion
    "Lanayru Gorge - Thunder Dragon's Reward",
```